sdk/search/Azure.Search/src/Generated/Operations/DocumentsRestClient.cs

```diff
@@ -246,14 +246,14 @@ internal HttpMessage CreateGetRequest(string key, IEnumerable<string> selectedFi
         /// <param name="selectedFields"> List of field names to retrieve for the document; Any field not retrieved will be missing from the returned document. </param>
         /// <param name="clientRequestId"> The tracking ID sent with the request to help with debugging. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public async ValueTask<Response<Components1fm1tvaSchemasFacetresultAdditionalproperties>> GetAsync(string key, IEnumerable<string> selectedFields, Guid? clientRequestId, CancellationToken cancellationToken = default)
+        public async ValueTask<Response<object>> GetAsync(string key, IEnumerable<string> selectedFields, Guid? clientRequestId, CancellationToken cancellationToken = default)
```

Comment:
I'm going to miss `Components1fm1tvaSchemasFacetresultAdditionalproperties`.

Reply:
It was the best :)